package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "eslint-plugin-eunomia",
+  "version": "0.0.0",
+  "description": "Home for all codified liting and style rules for The Agency",
+  "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/triplemint/eunomia.git"
+  },
+  "keywords": [
+    "lint",
+    "style",
+    "guide",
+    "rules"
+  ],
+  "author": "",
+  "license": "UNLICENSED",
+  "dependencies": {
+    "@thi.ng/diff": "^4.0.12",
+    "@types/estree": "^0.0.50",
+    "eslint": "^8.15.0",
+    "ts-node": "^10.7.0"
+  },
+  "bugs": {
+    "url": "https://github.com/triplemint/eunomia/issues"
+  },
+  "homepage": "https://github.com/triplemint/eunomia#readme",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
Shouldn't this conform to its own rules?

Reply:
Then the package name should definitely be changed.

Reply:
lol there is a reason for this too even though its dumb eslint requires plugins to be named eslint plugin-* so my thought was when the other stuff is added make this a mono repo so that this can make more sense

Reply:
What is the value of a monorepo over multiple repos?

Reply:
So you can go to a single repo to see all of our style rules